cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
+            type: 'string'
+        },
+        confirmPassword: {
+            describe: 'confirm the password',
+            demandOption: true,
+            type: 'string'
+        },
+        username: {
+            describe: 'and username',
+            demandOption: true,
+            type: 'string'
+        },
+        firstName: {
+            describe: 'your frist name',
+            demandOption: true,
+            type: 'string'
+        },
+        lastName: {
+            describe: 'your lastname',
+            demandOption: true,
+            type: 'string'
+        },
+        email: {
+            describe: 'your email',
+            demandOption: true,
+            type: 'string'
+        },
+        bio: {
+            describe: 'your biogaphy',
+            demandOption: true,
+            type: 'string'
+        },
+        website: {
+            describe: '...',
+            demandOption: true,
+            type: 'string'
+        },
+        location: {
+            describe: 'your actual location',
+            demandOption: true,
+            type: 'string'
+        },
+        picture: {
+            describe: '...',
+            demandOption: true,
+            type: 'string'
+        },Birthday:{
+            describe: 'User birthday',
+            type: 'string'
+        }
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        updateuser(argv)
+    }
+})
+
+//get post command
+y.scriptName("connectme")
+y.command({
+    command: 'get-post',
+    describe: 'gets a post',
+    builder: {
+        post: {
+            describe: 'Post to get',
+            demandOption: true,
+            type: 'string'
+        },
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        getpost(argv)
+    }
+})
+//Post command
+y.scriptName("connectme")
+y.command({
+    command: 'post',
+    describe: 'creates a post ',
+    builder: {
+        post: {
+            describe: 'Post name',
+            demandOption: true,
+            type: 'string'
+        },
+        canLike: {
+            describe: 'Possibility to like',
+            demandOption: true,
+            type: 'bool'
+        },
+       canUnlike: {
+           describe: 'Possibility to unlike',
+            demandOption: true,
+            type: 'bool'
+        },
+        likes: {
+            describe: 'Ammount of likes',
+            demandOption: true,
+            type: 'int'
+        },
+        tag: {
+            describe: 'Tag',
+            demandOption: true,
+            type: 'string'
+        },
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        createPost(argv)
+    }
+})
+//update post command
+y.scriptName("connectme")
+y.command({
+    command: 'update-post',
+    describe: 'updates a post',
+    builder: {
+        post: {
+            describe: 'Post name',
+            demandOption: true,
+            type: 'string'
+        },

```

Comment:
you need to specify the postId in order to update it, otherwise, how do we know which post to update?

Reply:
done